constraints.txt.in

```diff
@@ -37,7 +37,7 @@ PyYAML>=6.0.1,<6.1
 redis>=5.0,<5.1
 requests>=2.28.2,<2.29
 requests-oauthlib>=1.3.1,<1.4
-requests-retry-session>=0.1,<0.2
+requests-retry-session>=2.0,<2.1
```

Comment:
What's scary about 2.1?

Reply:
Nothing -- I just don't want it grabbing a newer minor version automatically. I figure newer patch versions should be fine, but we should decide if/when to bump the minor version.